### PR TITLE
docs: developers: how to build custom kernel

### PR DESCRIPTION
**Pull request by jcvenegas**
Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

File: docs/developers-clear-containers-install.md [hunk 1]
```diff
@@ -124,13 +124,41 @@ See [the upgrading document](upgrading.md) for further details.
...
+   The kernel latest kernel for Clear Linux can be found in
+   https://github.com/clearcontainers/linux/releases, It can be used to rebuild
+   and modify it as need. The example below demonstrates how to do this using the
+   osbuilder tooling.
+
+   ```bash
+	$ cd $GOPATH/src/github.com/clearcontainers/osbuilder
+
+	$ # Fetch latest kernel sources from github.com/clearcontainers/linux
+	$ make make kernel-src
```
Comment (mcastelino) [inline docs/developers-clear-containers-install.md]: This may be a typo `make make`

File: docs/developers-clear-containers-install.md [hunk 2]
```diff
@@ -124,13 +124,41 @@ See [the upgrading document](upgrading.md) for further details.
...
+
+	$ # Install the custom image
+	$ sudo install --owner root --group root --mode 0755 workdir/vmlinuz.container /usr/share/clear-containers/custom-vmlinuz
+	$ sudo install --owner root --group root --mode 0755 workdir/vmlinux.container /usr/share/clear-containers/custom-vmlinux
+
+	$ # Update the runtime configuration
+	$ # (note that this is only an example using default paths).
+	$ sudo sed -i.bak -e 's!^\(kernel = ".*"\)!# \1\nkernel = "/usr/share/clear-containers/custom-vmlinuz"!g' \
+	/usr/share/defaults/clear-containers/configuration.toml
+	 ```
```
Comment (mcastelino) [inline docs/developers-clear-containers-install.md]: We need to remember to add details for vmlinux too once we switch back from pc platform type.

File: docs/developers-clear-containers-install.md (hunk 1 above)
Comment (jcvenegas) [reply docs/developers-clear-containers-install.md]: oh thanks fixed.

File: docs/developers-clear-containers-install.md (hunk 2 above)
Comment (jcvenegas) [reply docs/developers-clear-containers-install.md]: @mcastelino added a note.

**Comment by jcvenegas** [comment]
@mcastelino fixed typos and added note.

File: docs/developers-clear-containers-install.md [hunk 3]
```diff
@@ -124,13 +124,43 @@ See [the upgrading document](upgrading.md) for further details.
...
+   ```
 For more details on the runtime's build system, run:
 
 ```bash
 $ make help
 ```
 
+5. Kernel
+
+   The kernel latest kernel for Clear Linux can be found in
```
Comment (jodh-intel) [inline docs/developers-clear-containers-install.md]: s/kernel latest kernel/latest kernel/

File: docs/developers-clear-containers-install.md [hunk 4]
```diff
@@ -124,13 +124,43 @@ See [the upgrading document](upgrading.md) for further details.
...
+   https://github.com/clearcontainers/linux/releases, It can be used to rebuild
+   and modify it as need. The example below demonstrates how to do this using the
```
Comment (jodh-intel) [inline docs/developers-clear-containers-install.md]: s/as need/as needed/

Comment (grahamwhaley) [reply docs/developers-clear-containers-install.md]: s/modify/modify a custom VM container kernel/

File: docs/developers-clear-containers-install.md [hunk 5]
```diff
@@ -124,13 +124,43 @@ See [the upgrading document](upgrading.md) for further details.
...
+   osbuilder tooling.
```
Comment (grahamwhaley) [inline docs/developers-clear-containers-install.md]: hotlink the [osbuilder](https://github.com/clearcontainers/osbuilder)

File: docs/developers-clear-containers-install.md [hunk 6]
```diff
@@ -124,13 +124,43 @@ See [the upgrading document](upgrading.md) for further details.
...
+
+   ```bash
+	$ cd $GOPATH/src/github.com/clearcontainers/osbuilder
+
+	$ # Fetch latest kernel sources from github.com/clearcontainers/linux
+	$ make kernel-src
+
+	$ # Optional modify kernel sources or config at $PWD/workdir/linux
```
Comment (grahamwhaley) [inline docs/developers-clear-containers-install.md]: s/Optional/Optionally/

**Comment by jcvenegas** [comment]
@grahamwhaley @jodh-intel thanks for the review, changes applied.
@iphutch could you take a look to this PR?

File: docs/developers-clear-containers-install.md [hunk 7]
```diff
@@ -124,13 +125,43 @@ See [the upgrading document](upgrading.md) for further details.
    $ sudo sed -i.bak -e 's!^\(image = ".*"\)!# \1 \
    image = "/usr/share/clear-containers/container.img"!g' \
    /usr/share/defaults/clear-containers/configuration.toml
-   
+   ```
 For more details on the runtime's build system, run:
```
Comment (iphutch) [inline docs/developers-clear-containers-install.md]: hmm, this should be indented along with the code block.

File: docs/developers-clear-containers-install.md [hunk 8]
```diff
@@ -124,13 +125,43 @@ See [the upgrading document](upgrading.md) for further details.
...
 
 ```bash
 $ make help
 ```
 
+5. Kernel
+
+   The latest kernel for Clear Linux can be found in
```
Comment (iphutch) [inline docs/developers-clear-containers-install.md]: Lets embed this link:
...can be found on [the releases page](https://github.com/clearcontainers/linux/releases). The Clear Linux kernel can ...

File: docs/developers-clear-containers-install.md (hunk 7 above)
Comment (jcvenegas) [reply docs/developers-clear-containers-install.md]: it is indented  with the code block above. It was missing to close it.

File: docs/developers-clear-containers-install.md (hunk 8 above)
Comment (jcvenegas) [reply docs/developers-clear-containers-install.md]: Added looks better !

**Comment by jodh-intel** [comment]
the jenkins 16.04 failure can be ignored: this is a doc-only change, it passed under 17.04 and the 16.04 issue is known to be a system problem.

**Pull request merged**